cmd/manager/main.go

```diff
@@ -387,4 +388,26 @@ func setupWebhook(mgr manager.Manager, certRotation certificates.RotationParams,
 		log.Error(err, "unable to create webhook", "webhook", "Elasticsearch")
 		os.Exit(1)
 	}
+
+	// wait for the secret to be populated in the local filesystem before returning
+	interval := time.Second * 1
+	timeout := time.Second * 30
+	keyPath := path.Join(mgr.GetWebhookServer().CertDir, "tls.crt")
```

Comment:
Nit: I think this should be `filepath.Join`. The `path` package is not OS-aware.

Reply:
Ah good catch, I actually started using filepath then switched because I got confused which was which 😞